packages/i/icu4c/xmake.lua

```diff
@@ -27,16 +36,14 @@ package("icu4c")
 
     on_install("windows", function (package)
         local configs = {path.join("source", "allinone", "allinone.sln"), "/p:SkipUWP=True", "/p:_IsNativeEnvironment=true"}
-        table.insert(configs, "/p:Configuration=" .. (package:debug() and "Debug" or "Release"))
-        table.insert(configs, "/p:Platform=" .. (package:is_arch("x64") and "x64" or "Win32"))
```

Comment:
You should not remove it now because xmake/dev is not yet released. This will break the current user's project.

Please wait until a new version of xmake is released before removing it.